testing/testing.gni

```diff
@@ -152,11 +155,16 @@ template("dart_snapshot_aot") {
 #     dart_main (required): The path to the main Dart file.
 template("dart_snapshot") {
   assert(defined(invoker.dart_main), "The main Dart file must be specified.")
+  assert(defined(invoker.dart_kernel_filename),
```

Comment:
Please document the new parameters `dart_kernel_filename` and `dart_elf_filename`

Reply:
Done